### PR TITLE
LKE-2577 Cosmos idOnly option

### PR DESCRIPTION
**Pull request by mzorkany**

### CODE REVIEW

**Review by yigitakbulut — APPROVED** [review]

File: src/api/GraphQuery/types.ts [hunk 1]
```diff
@@ -66,12 +66,14 @@ export interface EnumTemplate extends TemplateField<TemplateFieldType.ENUM> {
 export interface NodeTemplate extends TemplateField<TemplateFieldType.NODE> {
   options?: {
     categories?: string[];
+    serialize?: string;
```
Comment (pastahito) [inline src/api/GraphQuery/types.ts:69]: From what I understood in the ticket's last comment https://linkurious.atlassian.net/browse/LKE-2577?focusedCommentId=15570, the specs is `{idOnly?: boolean}`. What was the final decision?

Comment (mzorkany) [reply src/api/GraphQuery/types.ts:69]: @pastahito 

The original specs is to have a `serialize` option which accepts only the value `idOnly` as a string

The last comment in the ticket was from a discussion that i had with @davidrapin and @Diachk regarding the expected output when the `serialize` option is passed in the query template

Comment (davidrapin) [reply src/api/GraphQuery/types.ts:69]: @mzorkany can you point me to spec you are refering to?

Comment (mzorkany) [reply src/api/GraphQuery/types.ts:69]: @davidrapin 
https://linkurious.atlassian.net/browse/LKE-2577?focusedCommentId=15570

its written in the description of the ticket

Comment (pastahito) [reply src/api/GraphQuery/types.ts:69]: Nothing to changed, clarified by https://linkurious.atlassian.net/browse/LKE-2577?focusedCommentId=15927

**Review by pastahito — APPROVED** [review]

**Pull request merged**